pkg/grizzly/providers.go

```diff
@@ -45,8 +45,11 @@ func (r *Resource) MatchesTarget(targets []string) bool {
 	return false
 }
 
-// Resources represents a set of resources, by path
-type Resources map[string]Resource
+// ResourceList represents a set of named resources
+type ResourceList map[string]Resource
```

Comment:
```suggestion
type ResourceMap map[string]Resource
```

Also, `type Resources map[Handler]ResourceList` looks redundant.

Reply:
You're right that the naming is rubbish. Naming a map "List" sucks. The structure works well of a map by handler containing a map by name of resources - I just need to work out the naming.